doc/constitution.md

```diff
@@ -155,7 +155,7 @@ After the nomination deadline, the SC can prevent a nominee from becoming a cand
 
 #### Procedure
 
-The election is done using a [proportional representation mode from Condorcet Internet Voting Service (CIVS)](http://web.archive.org/web/20240412235900/https://civs1.civs.us/proportional.html). The recommendation is to use [CIVS](https://civs1.civs.us/) itself, but if CIVS is unavailable, a reasonable alternative proportional representation election implementation is also permitted.
+The election is done using a [proportional representation mode from Condorcet Internet Voting Service (CIVS)](http://web.archive.org/web/20240412235900/https://civs1.civs.us/proportional.html). The recommendation is to use [CIVS](https://civs1.civs.us/) itself, but a reasonable alternative proportional representation election implementation is also permitted.
```

Comment:
```suggestion
The election is done using a propotional representation oriented tallying system based on ranked ballots. The recommendation is to use [proportional representation mode from Condorcet Internet Voting Service (CIVS)](http://web.archive.org/web/20240412235900/https://civs1.civs.us/proportional.html), but a reasonable alternative proportional representation election implementation is also permitted.
```

Reply:
How about even removing CIVS as a recommendation:

```suggestion
The election is done using a propotional representation oriented tallying system based on ranked ballots. One option is to use [proportional representation mode from Condorcet Internet Voting Service (CIVS)](http://web.archive.org/web/20240412235900/https://civs1.civs.us/proportional.html), but a reasonable alternative is also permitted.
```

Reply:
(So that a native propotional tallying system of a neutral platform can be used, e.g. for OpaVote there is Meek-STV)

Reply:
I do not see any problems with the no-recommendation wording.